.github/workflows/build.yaml

```
@@ -0,0 +1,73 @@
+name: Build for slsa
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v*'
```

You don't want to build everytime a pull request is opened, reopened or a changed pushed... https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Same as for the tagging, let's only run this when the TaskWeaver team wants to manually do a release:
```suggestion
on:
  release:
    types: [published]
```